src/main/gen/software/amazon/smithy/plugin/language/SmithyLexer.java

```diff
@@ -0,0 +1,1005 @@
+/* The following code was generated by JFlex 1.7.0-1 tweaked for IntelliJ platform */
+
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.plugin.language;
+
+import com.intellij.lexer.FlexLexer;
+import com.intellij.psi.tree.IElementType;
+
+import static com.intellij.psi.TokenType.BAD_CHARACTER;
+import static com.intellij.psi.TokenType.WHITE_SPACE;
+import static software.amazon.smithy.plugin.language.psi.SmithyTypes.*;
+
+/**
+ * This class is a scanner generated by JFlex
+ * from the specification file <tt>Smithy.flex</tt>
+ */
+
```

Comment:
There's an issue with the Lexer generator vended in the Grammar-Kit plugin that results in this extra space after the doc.